modules/node_modules/@ncigdc/uikit/Dropdown.js

```diff
@@ -35,7 +37,7 @@ const Dropdown = ({
         onMouseUp={mouseUpHandler}
         onClick={() => setTimeout(() => setActive(false))}
       >
-        {Children.map(children, child =>
+        {Children.map(_.filter(children, React.isValidElement), child =>
```

Comment:
That function is quite robust and does a lot under the hood :)

https://github.com/facebook/react/blob/6810627a91f2020caee7fecf21580187dc6ef0fc/src/isomorphic/children/ReactChildren.js#L161-L168

https://github.com/facebook/react/blob/6810627a91f2020caee7fecf21580187dc6ef0fc/src/shared/utils/traverseAllChildren.js#L98-L192